README.md

```diff
@@ -16,6 +16,15 @@ git clone git@github.com:ModellingWebLab/deployment.git
 cd deployment
 git submodule update --init
 ```
+### Running in a folder
+If you're running weblab in a folder e.g. http://myhost.com/weblab/ then edit `roles/django/templates/deployed.j2` and add the following lines with the correct folder name e.g.:
+
+```
+FORCE_SCRIPT_NAME = '/weblab/'
+STATIC_URL = '/weblab/static/'
+LOGIN_REDIRECT_URL = '/weblab/'
+LOGOUT_REDIRECT_URL = '/weblab/'
+```
 
```

Comment:
```suggestion

Note that you will need to set up the appropriate proxying *outside* the Web Lab deployment so that requests to your sub-URL come through to the nginx server used by Web Lab. Such configuration is outside the scope of this guide.

```